Dockerfile

```diff
@@ -1,13 +1,12 @@
-FROM library/node:latest
+FROM node:latest
 
-RUN mkdir /src
 WORKDIR /src
 
-COPY package.json .
-RUN npm install
+COPY package.json package-lock.json ./ 
+RUN npm ci
 
 COPY . .
```

Comment:
Why COPY twice? (Not saying it is not needed, but I don't see the reason.) Wouldn't the following work?
```
COPY . .
RUN npm ci
RUN npm run build
```
or even shorter:
```
COPY . .
RUN npm ci && npm run build
```
(Single COPY)

Reply:
@pdcastro Thought we follow best practice here of incremental builds. 

> Using this separation, if there are no changes in package.json or package-lock.json then the cache will be used for the layer generated by the RUN npm ci instruction. This means that when you edit your application source and rebuild, the dependencies won’t be redownloaded which saves time

Reference: https://www.docker.com/blog/speed-up-your-development-flow-with-these-dockerfile-best-practices/

Reply:
That's a good one! 👍

Reply:
Talking of Docker layer caching, I've noticed that `docker build .` for this repo will use the cache for all layers (will not rebuild the image) if the external markdown files changed (from repos like the supervisor, CLI, etc), but nothing else changed. I believe (not 100% sure) that this was a pre-existing issue before this PR, even when the `postinstall` script was used. I am not sure that this is a problem in practice (I suspect it is not), but if it was, stackoverflow has suggestions for forcing cache invalidation using ARG and RUN:
https://stackoverflow.com/questions/35134713/disable-cache-for-specific-run-commands